cpu/stm32/cpu_common.c

```diff
@@ -60,6 +64,7 @@ uint32_t periph_apb_clk(uint8_t bus)
     else {
```

Comment:
would

```suggestion
#ifdef CLOCK_APB2
    else {
```

also work?

Reply:
probably, but I don't know if it will be optimized the same.

Reply:
How about

```C
uint32_t periph_apb_clk(uint8_t bus)
{
#ifdef CLOCK_APB2
    if (bus == APB2) {
        return CLOCK_APB2;
    }
#else
    (void) bus;
#endif

    return CLOCK_APB1;
}
```

Reply:
It's not so simple because there's also the APB12 bus on L4/WB/G4.

Reply:
Yea but right now we have

```C
#if defined (CPU_FAM_STM32L4) || defined(CPU_FAM_STM32WB) || \
    defined(CPU_FAM_STM32G4)
    else if (bus == APB12) {
        return CLOCK_APB1;
    }
#endif
```

so in any case when `bus != APB2`, `CLOCK_APB1` is returned.

Reply:
You are right. I changed it that way.